api/k8s/v1/k8s.proto

```diff
@@ -132,6 +132,14 @@ message DeletePodRequest {
 message DeletePodResponse {
 }
 
+// Annotation value is wrapped in the message field to make it nullable.
+// Setting AnnotationValue to nil means the annotation shouldn't exist or
+// should be deleted.
+message AnnotationValue {
```

Comment:
google provides the StringValue wrapper in core proto for this purpose but maybe it's still good to have it's own object for documentation purposes. just thinking out loud.